src/checkpoints/checkpoints.cpp

```diff
@@ -167,38 +167,17 @@ namespace cryptonote
       ADD_CHECKPOINT(1000000, "46b690b710a07ea051bc4a6b6842ac37be691089c0f7758cfeec4d5fc0b4a258");
```

Comment:
Testnet checjpoint ought to be removed too, no ?

Reply:
ok

Reply:
ok